deepmd/pt/model/task/denoise.py

```
@@ -1,137 +1,568 @@
+            "use_aparam_as_mask": self.use_aparam_as_mask,
+            "spin": None,
+        }
+
+    def deserialize(self) -> "DenoiseNet":
+        data = data.copy()
+        variables = data.pop("@variables")
+        cell_nets = data.pop("cell_nets")
+        coord_nets = data.pop("coord_nets")
+        obj = cls(**data)
+        for kk in variables.keys():
+            obj[kk] = to_torch_tensor(variables[kk])
```

Ruff (0.8.2)

Undefined name `to_torch_tensor`

(F821)